src/api/admin/types.ts

```diff
@@ -646,7 +646,7 @@ export type PeerMetaInfoResponse = Record<
  */
 export interface PeerMetaInfo {
   meta_value: string;
-  expires_at: Timestamp;
+  expires_at: Timestamp | undefined;
```

Comment:
```suggestion
  expires_at?: Timestamp;
```

Reply:
I had it like this first but then changed it to `| undefined` because I thought it's not an input parameter type but a type you get in a response so I thought the question mark is less appropriate in such a case. And I saw other usages of `| undefined` for output types as well in the code base. But I changed it to `?` now, I'm not strongly attached.

Reply:
In fact, the correct type would probably be `Timestamp | null` since I think a `None` value [gets deserialized](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=a4ecc54186a67b2c4237d88dcf2abeaa) as `null`. The question mark would indicate that a field does not need to be there at all but I think in this case it will always be there.